pages/vi/vi-github-and-markdown.md

```diff
@@ -58,15 +58,17 @@ First, you will need to find the correct GitHub repository to fork. Forking is e
 
 ### Go to Settings and rename your repository
 
-After you fork the repository, you will be on your page: &lt;username&gt;/open-learning-exchange.github.io. Then, you should go to Settings at the top middle of the page and rename your repository to your GitHub username. See example below.
+After you fork the repository, you will be on your page: &lt;username&gt;/open-learning-exchange.github.io. Then, you should go to Settings at the top middle of the page and rename your repository to your GitHub username.  See example below.
 
 ![Renaming Repository](images/vi-rename-repository.png)
 
+Check [FAQ Q16](http://open-learning-exchange.github.io/#!pages/vi/vi-faq.md) if you already have a repository named YourUserName.github.io.
```

Comment:
- it would be better to use relative links whenever possible, please see #515
- could you add Q16's id `#16...` to the end of the link so it goes straight to Q16?